lib/new-config-manager.js

```diff
@@ -78,17 +70,23 @@ const getAllCryptoCurrencies = (config) => {
 const getNotifications = (cryptoCurrency, machine, config) => {
   const notifications = fromNamespace(namespaces.NOTIFICATIONS)(config)
 
-  const cryptoFilter = _.matches({ cryptoCurrency })
-  const withCryptoBalance = resolveOverrides(notifications, cryptoFilter, notifications.cryptoBalanceOverrides, 'cryptoBalanceOverrides')
+  const smsSettings = fromNamespace('sms', notifications)
+  const emailSettings = fromNamespace('email', notifications)
+
+  const cryptoOverride = _.filter(o => o.cryptoCurrency === cryptoCurrency)(notifications.cryptoBalanceOverrides)[0]
+  const cryptoHighBalance = _.getOr(notifications.cryptoHighBalance, ['highBalance'], cryptoOverride) // cryptoOverride ? cryptoOverride.highBalance : notifications.cryptoHighBalance
+  const cryptoLowBalance = _.getOr(notifications.cryptoLowBalance, ['lowBalance'], cryptoOverride) // cryptoOverride ? cryptoOverride.lowBalance : notifications.cryptoLowBalance
+  const cryptoSettings = { cryptoHighBalance, cryptoLowBalance }
 
-  const fiatFilter = _.matches({ machine })
-  const withFiatBalance = resolveOverrides(withCryptoBalance, fiatFilter, withCryptoBalance.fiatBalanceOverrides, 'fiatBalanceOverrides')
+  const fiatOverride = _.filter(o => o.machine === machine)(notifications.fiatBalanceOverrides)[0]
+  const fiatBalanceCassette1 = _.getOr(notifications.fiatBalanceCassette1, ['fiatBalanceCassette1'], fiatOverride)
+  const fiatBalanceCassette2 = _.getOr(notifications.fiatBalanceCassette2, ['fiatBalanceCassette2'], fiatOverride)
+  const fiatSettings = { fiatBalanceCassette1, fiatBalanceCassette2 }
 
-  const withSms = fromNamespace('sms', withFiatBalance)
-  const withEmail = fromNamespace('email', withFiatBalance)
+  const id = cryptoOverride ? cryptoOverride.id : fiatOverride ? fiatOverride.id : null
```

Comment:
Hmm, I know I told this is important to preserve, but this is only preserving for cryptoOverrides, not fiatOverrides. You can take this out, we can export when needed.